DataFormats/PatCandidates/interface/TriggerObjectStandAlone.h

```diff
@@ -33,8 +34,10 @@ namespace pat {
       /// Keeping the old names of the data members for backward compatibility,
       /// although they refer only to HLT objects.
 
-      /// Vector of labels of all HLT filters or names od L1 conditions the trigger objects has been used in
-      std::vector< std::string > filterLabels_;
+      /// Vector of labels of all HLT filters or names of L1 conditions the trigger objects has been used in
+      mutable std::vector< std::string > filterLabels_;
+      mutable std::vector< uint16_t >    filterLabelIndices_; 
```

Comment:
the original design was to have both possibly filled with the first being a fallback when the name cannot be mapped and at the same time acting as the "unpacked" container.
probably we do not need this flexibility and we can stick to one mutable and one persistent but a decision has not been made yet.